src/Public.Api/Parcel/BackOffice/ParcelBackOfficeController-DetachAddress.cs

```diff
@@ -0,0 +1,87 @@
+    using System.Threading.Tasks;
+    using Be.Vlaanderen.Basisregisters.Api.Exceptions;
+    using Common.Infrastructure;
+    using Common.Infrastructure.Extensions;
+    using Infrastructure;
+    using Infrastructure.Swagger;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc.Infrastructure;
+    using ParcelRegistry.Api.BackOffice.Abstractions.Requests;
+    using ParcelRegistry.Api.Legacy.Parcel.Responses;
+    using RestSharp;
+    using Swashbuckle.AspNetCore.Annotations;
+    using Swashbuckle.AspNetCore.Filters;
+    using ProblemDetails = Be.Vlaanderen.Basisregisters.BasicApiProblem.ProblemDetails;
+
+    public partial class ParcelBackOfficeController
+    {
+        public const string DetachAddressParcelRoute = "percelen/{objectId}/acties/adresontkoppelen";
+
+        /// <summary>
+        /// Ontkoppel een perceel van een adres.
+        /// </summary>
+        /// <param name="objectId">Identificator van het perceel.</param>
+        /// <param name="actionContextAccessor"></param>
+        /// <param name="problemDetailsHelper"></param>
+        /// <param name="ifMatch">If-Match header met ETag van de laatst gekende versie van het perceel (optioneel).</param>
+        /// <param name="toggle"></param>
+        /// <param name="cancellationToken"></param>
+        /// <response code="202">Als het ticket succesvol is aangemaakt.</response>
+        /// <response code="400">Als uw verzoek foutieve data bevat.</response>
+        /// <response code="404">Als het adres niet gevonden kan worden.</response>
+        /// <response code="406">Als het gevraagde formaat niet beschikbaar is.</response>
+        /// <response code="412">Als de If-Match header niet overeenkomt met de laatste ETag.</response>
+        /// <response code="429">Als het aantal requests per seconde de limiet overschreven heeft.</response>
+        /// <response code="500">Als er een interne fout is opgetreden.</response>
+        /// <returns></returns>
+        [ApiOrder(ApiOrder.Parcel.Edit + 2)]
+        [ProducesResponseType(StatusCodes.Status202Accepted)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status404NotFound)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status412PreconditionFailed)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status429TooManyRequests)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        [SwaggerResponseHeader(StatusCodes.Status202Accepted, "location", "string", "De URL van het aangemaakte ticket.")]
+        [SwaggerResponseHeader(StatusCodes.Status202Accepted, "x-correlation-id", "string", "Correlatie identificator van de response.")]
+        [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamples))]
```

Comment:
Add the request example, like here https://github.com/Informatievlaanderen/public-api/blob/main/src/Public.Api/BuildingUnit/BackOffice/BuildingUnitBackOfficerController-Plan.cs#L43